Header header

string[] object_name
string[] hand_name
---
int64 w_flag
